### PR TITLE
deps: update bigtable emulator versions

### PR DESCRIPTION
**Pull request by rkaregar**

### CODE REVIEW

**Review by igorbernstein2 — APPROVED** [review]

**Comment by rkaregar** [comment]
Hi @noahdietz , I noticed you had added a comment on a previous similar PR, so was wondering if you could take a look at the current one as well.

The `bigtable_conformance (1.20, bigtable)` and `vet` steps are failing with `/google-cloud-go/bigtable/go.mod:5: unknown directive: toolchain` error, which seems to come from the new `toolchain go1.22.5` line in the `go.mod` file. This was automatically updated when I ran the `go get -u ./...` command. I was wondering if it's best to merge the PR with that line in, or if it should be removed.

File: go.work [hunk 1]
```diff
@@ -1,4 +1,6 @@
-go 1.20
+go 1.21
```
Comment (bhshkh) [inline go.work:1]: https://github.com/googleapis/google-cloud-go?tab=readme-ov-file#go-versions-supported 

Minimum version currently is go 1.20. Please do not merge this PR.

Comment (bhshkh) [reply go.work:1]: Minimum version is go 1.21 now

**Comment by codyoss** [comment]
Closing as we can not accept this change as is.

**Pull request closed without merging**